src/stp/new_plays/PenaltyThem.cpp

```diff
@@ -0,0 +1,64 @@
+//
+// Created by timovdk on 4/28/20.
+//
+
+#include <stp/invariants/game_states/PenaltyThemGameStateInvariant.h>
+#include <stp/new_plays/PenaltyThem.h>
+#include <stp/new_roles/Halt.h>
+#include <stp/new_roles/PenaltyKeeper.h>
+
+namespace rtt::ai::stp::play {
+
+PenaltyThem::PenaltyThem() : Play() {
+    startPlayInvariants.clear();
+    startPlayInvariants.emplace_back(std::make_unique<invariant::PenaltyThemGameStateInvariant>());
+
+    keepPlayInvariants.clear();
+    keepPlayInvariants.emplace_back(std::make_unique<invariant::PenaltyThemGameStateInvariant>());
+
+    roles = std::array<std::unique_ptr<Role>, stp::control_constants::MAX_ROBOT_COUNT>{std::make_unique<role::PenaltyKeeper>(role::PenaltyKeeper("keeper")),
+                                                                                       std::make_unique<role::Halt>(role::Halt("halt_0")),
```

Comment:
Should these be halts? I think they also need to move out of the defense area, right? And behind the ball?